.gitattributes

```diff
@@ -0,0 +1,5 @@
+.gitattributes export-ignore
+.gitignore export-ignore
+.travis.yml export-ignore
+tests/ export-ignore
+*.sh eol=lf
```

Comment:
`.sh` is relevant to linux only, and nobody on linux is using windows line endings?